.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+updates:
+  - package-ecosystem: gradle
+    directory: /
+    reviewers:
+      - Nikita-Smirnov-Exactpro
+      - OptimumCode
+    labels:
+      - dependencies
```

Comment:
note: you need to create a label [dependencies](https://github.com/th2-net/th2-mstore/labels/dependencies) before dependabot can use it. I have created it for this repo